R/algorithm.R

```diff
@@ -40,22 +40,22 @@ create_SPC_auto_limits_table <- function(data,
       dplyr::mutate(y = dplyr::if_else(is.nan(y) | is.infinite(y), as.numeric(NA), y))
   }
   
-  #set counter to zero
-  counter <- 0
+  #set counter to one
+  counter <- 1
   
   #[1] see whether there are enough data points to form one period
   if(enough_data_for_new_period(data, periodMin, counter)){
     
     #[2]
     limits_table <- form_calculation_and_display_limits(data = data, 
                                                         periodMin = periodMin, 
-                                                        counter = counter, 
+                                                        counter_at_period_start = counter, 
```

Comment:
This one was a separate thing, I think just an old argument name. I think this will have been working "by accident" because when inside the function R cannot find `counter`, it would look up the call stack to the next environment and find `counter` there - which would give it the correct value. But obviously better to use the new argument name.